lib/FusionInventory/Agent/Tools/Generic.pm

```diff
@@ -12,6 +12,7 @@ use FusionInventory::Agent::Tools;
 
 our @EXPORT = qw(
     getDmidecodeInfos
+    getIpmiFru
```

Comment:
getIpmiFru is too specific and should only be seen in related module.

Reply:
I don't think so. Same as _dmidecode_, _ipmitool fru_ returns information for multiple devices. It's just not used anywhere **yet**. But if you insist, i'll put the function into the related module.